src/custom-elements/src/components/carousel/carousel.tsx

```diff
@@ -16,6 +16,14 @@ export class Carousel {
    * To render Carousel with dot controllers
    */
   @Prop() dots: boolean;
+  /**
+   * To render Carousel with pagination indicators
+   */
+  @Prop() autoplay: boolean;
+   /**
+   * To render Carousel with pagination indicators
+   */
+  @Prop() delay: number;
```

Comment:
Gives a default delay in case the user does not set this property.

```suggestion
  @Prop() delay: number = 3000;
```